<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newMap</class>
 <widget class="QWidget" name="newMap">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>227</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Map</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>161</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="formatLabel">
      <property name="text">
       <string>label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="formatListWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QLabel" name="layerLabel">
      <property name="text">
       <string>label</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QSlider" name="layerSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="geometryLabel">
      <property name="text">
       <string>label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sizeLineEdit">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>211</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QLabel" name="projectNameLabel">
      <property name="text">
       <string>label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="saveBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
